client/src/services/upload/fileService.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    upload(file) {
+        let formData = new FormData();
+
+        formData.append('file', file);
+
+        //TODO change to the real endpoint
```

Comment:
you do not need these todos, you anyway will add endpoints, there is no other way to make it work